src/op/compute_op.cc

```diff
@@ -329,7 +329,10 @@ Stmt MakeComputeStmt(const ComputeOpNode* self,
         n.main_nest.begin() + n.num_common_loop + 1, n.main_nest.end());
     provide = op::Substitute(provide, n.main_vmap);
     provide = MergeNest(reduce, provide);
-    return MergeNest(common, Block::make(init, provide));
+    if (del_trivial_loop)
```

Comment:
because it generates in compatible behavior, let us rename del_trivial_loop to be debug_keep_trivial_loop